src/System.Configuration/ref/System.Configuration.cs

```diff
@@ -3,37 +3,517 @@
+        public override int GetHashCode() { throw null; }
+        protected override bool IsModified() { throw null; }
+        protected override void Reset(System.Configuration.ConfigurationElement parentElement) { }
+        protected override void ResetModified() { }
+        protected override bool SerializeToXmlElement(System.Xml.XmlWriter writer, string elementName) { throw null; }
+        protected override void Unmerge(System.Configuration.ConfigurationElement sourceElement, System.Configuration.ConfigurationElement parentElement, System.Configuration.ConfigurationSaveMode saveMode) { }
+    }
+    public partial class SingleTagSectionHandler : System.Configuration.IConfigurationSectionHandler
+    {
+        public SingleTagSectionHandler() { }
+        public virtual object Create(object parent, object context, System.Xml.XmlNode section) { throw null; }
+    }
+    public enum SpecialSetting
+    {
+        ConnectionString = 0,
+        WebServiceUrl = 1,
+    }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Class | System.AttributeTargets.Property)]
+    public sealed partial class SpecialSettingAttribute : System.Attribute
+    {
+        public SpecialSettingAttribute(System.Configuration.SpecialSetting specialSetting) { }
+        public System.Configuration.SpecialSetting SpecialSetting { get { throw null; } }
+    }
+    public sealed partial class UriSection : System.Configuration.ConfigurationSection
+    {
+        public UriSection() { }
+        [System.Configuration.ConfigurationPropertyAttribute("idn")]
+        public System.Configuration.IdnElement Idn { get { throw null; } }
+        [System.Configuration.ConfigurationPropertyAttribute("iriParsing")]
+        public System.Configuration.IriParsingElement IriParsing { get { throw null; } }
+        protected override System.Configuration.ConfigurationPropertyCollection Properties { get { throw null; } }
+        [System.Configuration.ConfigurationPropertyAttribute("schemeSettings")]
+        public System.Configuration.SchemeSettingElementCollection SchemeSettings { get { throw null; } }
+    }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Property)]
+    public sealed partial class UserScopedSettingAttribute : System.Configuration.SettingAttribute
+    {
+        public UserScopedSettingAttribute() { }
+    }
+    public sealed partial class UserSettingsGroup : System.Configuration.ConfigurationSectionGroup
+    {
+        public UserSettingsGroup() { }
+    }
+
     public sealed partial class AppSettingsSection : System.Configuration.ConfigurationSection
     {
         public AppSettingsSection() { }
         [System.Configuration.ConfigurationPropertyAttribute("file", DefaultValue = "")]
         public string File { get { throw null; } set { } }
-        protected internal override System.Configuration.ConfigurationPropertyCollection Properties { get { throw null; } }
```

Comment:
Also, if GenAPI was emitting this you could fix GenAPI, or file an issue in BuildTools.